frontend/src/features/env/PublicFeedbackModal.tsx

```diff
@@ -80,6 +91,28 @@ export const PublicFeedbackModal = ({
     publicSwitchEnvMutation.mutate()
   }, [publicSwitchEnvMutation])
 
+  const checkboxInputName = 'attachmentType'
+
+  const CHECKBOX_OTHERS_INPUT_VALUE =
+    '!!FORMSG_INTERNAL_CHECKBOX_OTHERS_VALUE!!'
+  const ATTACHMENT_TYPE_OPTIONS = ['PDF', 'JPEG', 'PNG', 'ZIP']
+
+  const othersValidationRules = useMemo(
+    () => ({
+      validate: (value?: string | string[]) => {
```

Comment:
When would value be `string[]`?

Reply:
`string` only oops, modified [here](https://github.com/opengovsg/FormSG/pull/5582/commits/ce1650222cf66faacd5dcbdab9d5cbb5e074d27d#diff-b2271c6e3f57c5e2f1759290092b864d15724f49237d1c5348bfc51ee5eac5a1R102-R109)